src/pinejs-session-store/pinejs-session-store.ts

```diff
@@ -0,0 +1,133 @@
+import * as _ from 'lodash'
+import { Store } from 'express-session'
+import * as permissions from '../sbvr-api/permissions'
+import { api, AnyObject } from '../sbvr-api/sbvr-utils';
+import { Config } from '../config-loader/config-loader';
+
+const sessionModel = `
+	Vocabulary: session
+
+	Term:       session id
+		Concept Type: Short Text (Type)
+	Term:       data
+		Concept Type: JSON (Type)
+	Term:       expiry time
+		Concept type: Date Time (Type)
+
+	Term:       session
+		Database ID Field: session id
+		Reference Scheme: session id
+
+	Fact type:  session has data
+		Necessity: Each session has exactly 1 data
+	Fact type:  session has session id
+		Necessity: Each session has exactly 1 session id
+		Necessity: Each session id is of exactly 1 session
+	Fact type:  session has expiry time
+		Necessity: Each session has at most 1 expiry time
+`
+
+export = class PinejsSessionStore extends Store {
+	get = ((sid, callback) => {
+		api.session.get({
```

Comment:
Don’t you need to return the promise as well?

Reply:
Nope, express-session only supports a callback interface (the typings would complain if I did return the promise)